docs/learn/learn-staking.md

```diff
@@ -489,170 +445,11 @@ Polkadot allows some validators to be disabled, but if the number of disabled va
-{{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} for staking, your tokens dilute over time.
-
-:::
-
-The ideal staking rate on {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} is designed
-to be dynamic and is adjusted based on the number of parachain slots. As the number of parachain
-slots increase, the ideal staking rate decreases.
-
-:::note
-
-The current staking rate on Polkadot still assumes the absence of parachains, with the suggested
-ideal staking rate of 75%. You can track the progress on the issue to adjust it
-[here](https://github.com/paritytech/polkadot/pull/5872). This has already been adjusted on Kusama.
-At the time of updating this section, the ideal staking rate on Kusama was 50.5% when there were 49
-parachain slots. When the number of slots goes to 60, the ideal staking rate will be 45%.
-[Here](https://github.com/paritytech/polkadot/blob/master/runtime/kusama/src/lib.rs#L535) is the
-code for reference.
-
-:::
-
-If the amount of tokens staked goes below the ideal rate, then staking rewards for nominators go up
-incentivizing to stake more tokens on the network. On the contrary, if it goes above the ideal rate,
-staking rewards drop. This is a result of the change in the percentage of staking rewards that go to
-the Treasury.
-
-![staking](../assets/NPoS/staking-rate-with-parachains.png)
-
-<p style={{textAlign:"center"}}>Source: <a href="https://w3f-research.readthedocs.io/en/latest/polkadot/overview/2-token-economics.html">Research - Web3 Foundation</a></p>
-
-- **x-axis**: Proportion of {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} staked
-- **y-axis**: Inflation, annualized percentage
-- **Blue line**: Annual inflation rate of NPoS, i.e. total amount of tokens minted to pay validators
-  and nominators.
-- **Green line**: Annual rewards rate for stakers. For instance, 0.2 corresponds to 20% of annual
-  returns on the staked tokens.
-
-You can determine the staking rewards by checking the staking overview on
-[Polkadot-JS Apps](https://polkadot.js.org/apps/#/staking).
-
-The above chart shows the inflation model of the network. Depending on the staking participation,
-the distribution of the inflation to validators/nominators versus the treasury will change
-dynamically to provide incentives to participate (or not participate) in staking.
-
-For instance, assuming that the ideal staking rate is 50%, all of the inflation would go to the
-validators/nominators if 50% of all KSM / DOT are staked. Any deviation from the 50% - positive or
-negative - sends the proportional remainder to the treasury and effectively reduces staking rewards.
+For more on chilling, see the "[How to Chill][]" page on this wiki.
 
-For those who are interested in knowing more about the design of inflation model for the network,
-please see
-[here](https://w3f-research.readthedocs.io/en/latest/polkadot/overview/2-token-economics.html).
+## Why and why not staking?
```

Comment:
Is "Why and why not to stake?" more natural?

Reply:
Changed